tests/conftest.py

```diff
@@ -70,7 +70,11 @@ def ensurerepo(lakefs_client: LakeFSClient) -> str:
     if _TEST_REPO in reponames:
         logger.info(f"Test repository {_TEST_REPO!r} already exists.")
     else:
-        storage_config = lakefs_client.config_api.get_storage_config()
+        # TODO (n.junge): Remove when the required `lakefs-client` version is at least v0.111.0.
+        if lakefs_client_version < (0, 111, 0):
```

Comment:
This is a bit brittle, you might want to replace it with [`packaging.version.Version` from the standard library](https://packaging.pypa.io/en/stable/version.html):

```suggestion
        if lakefs_client_version < Version("0.111.0"):
```